utils/cluster_stability.py

```diff
@@ -132,7 +132,7 @@ def return_clustering_solutions(electrode_num):
 	fig.suptitle(f'Electrode {electrode_num:02} - {clust_num} clusters')
 	fig.tight_layout()
 	fig.subplots_adjust(top=0.95)
-	fig.savefig(f'./Plots/{electrode_num:02}/clusters{clust_num:02}/' +\
+	fig.savefig(f'./Plots/{electrode_num:02}/clusters{clust_num}/' +\
```

Comment:
Revert back - zero-padding is to support higher numbers of clusters and keep the styling uniform.

Reply:
Reverted to zero-padding cluster directory paths